modules/script.js

```diff
@@ -0,0 +1,94 @@
+import Books from './displayBook.js';
+import { DateTime } from './luxon/src/luxon.js';
+
+const allBooks = new Books([]);
+
+function addToLocalStorage() {
+  const stringifyArray = JSON.stringify(allBooks.bookArray);
+  localStorage.setItem('storedBooks', stringifyArray);
+}
+
+const clockElement = document.getElementById('date');
+const clock = () => {
+  clockElement.textContent = DateTime.now().toISO();
+};
+setInterval(clock, 1000);
+
+function displayBook() {
+  const addedBooks = document.getElementById('list');
+  addedBooks.innerHTML = '';
+  for (let i = 0; i < allBooks.bookArray.length; i += 1) {
+    const container = document.createElement('div');
+    container.classList.add('book');
+    addedBooks.appendChild(container);
+
+    const container02 = document.createElement('div');
+    container02.classList.add('titleAuthor');
+    container.appendChild(container02);
+
+    const bookDetails = document.createElement('p');
+    bookDetails.classList.add('title');
+    bookDetails.textContent = `"${allBooks.bookArray[i].title}" by ${allBooks.bookArray[i].author}`;
+    container02.appendChild(bookDetails);
+
+    const removeButton = document.createElement('button');
+    removeButton.classList.add('remove');
+    removeButton.textContent = 'Remove';
+
+    removeButton.onclick = () => {
+      allBooks.booksFilter(allBooks.bookArray[i]);
+      addToLocalStorage();
+      displayBook();
+    };
+    container02.appendChild(removeButton);
+  }
```

Comment:
- [ ] good job refactoring :clap: , Please notice that one of the main goals of this project is to refactor it to use ES6 syntax, which uses arrow functions instead, I suggest we change all the functions to be arrow functions :pray: